react-app/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
```

Comment:
Can you update this readme file on how to integrate sunbird pdf player with react js

Reply:
Updated with changes